alpa/device_mesh.py

```diff
@@ -1123,6 +1217,61 @@ def delete(self):
     def flush(self):
         self._npy_value = None
 
+    #======== distributed save/load ========== START#
+    
+    def dist_view(self):
```

Comment:
Use `self.device_mesh.get_remote_buffers(self.remote_buffers, batching=True)` and compare the numerical values in unit test instead of comparing strings

Reply:
Is this comment resolved?